src/library/Uno.Material/Styles/Controls/v2/Slider.xaml

```diff
@@ -96,9 +97,10 @@
 			<x:Double x:Key="SliderThumbWidth">20</x:Double>
 			<x:Double x:Key="SliderThumbHeight">20</x:Double>
 			<x:Double x:Key="SliderThumbCornerRadius">10</x:Double>
-			<StaticResource x:Key="SliderTrackCornerRadius" ResourceKey="ControlCornerRadius" />
+			<Thickness x:Key="SliderThumbMargin">-1</Thickness>
```

Comment:
Not sure if negative margin is the way to go here as it is affecting the thumb position to be perfectly centered for the proper value. Will need to be verified